### PR TITLE
[frontend] Add Base Image Collections

### PR DESCRIPTION
**Pull request by szakhlypa**
Adds forms and logic to create, update and delete Base Image Collections.

System model is required to create a new Base Image Collection.
bic_no_system_model

Create, filter and delete Base Image Collection

https://user-images.githubusercontent.com/26611935/205865917-dc9df75f-8e38-48c7-a165-775c0346650c.mp4

Update Base Image Collection

https://user-images.githubusercontent.com/26611935/205866068-11460ac3-75bf-43c8-a30b-45f916a5c28e.mp4

### CODE REVIEW

File: frontend/src/pages/BaseImageCollections.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2022 SECO Mind Srl
```
Comment (Pavinati) [inline frontend/src/pages/BaseImageCollections.tsx:4]: 2023

File: frontend/src/forms/UpdateBaseImageCollection.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2022 SECO Mind Srl
```
Comment (Pavinati) [inline frontend/src/forms/UpdateBaseImageCollection.tsx:4]: 2023

File: frontend/src/pages/BaseImageCollection.tsx [hunk 3]
```diff
@@ -0,0 +1,330 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2022 SECO Mind Srl
```
Comment (Pavinati) [inline frontend/src/pages/BaseImageCollection.tsx:4]: 2023

**Review by Pavinati — CHANGES_REQUESTED** [review]

**Review by Pavinati — APPROVED** [review]

File: frontend/src/forms/CreateBaseImageCollection.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    mode: "onTouched",
+    defaultValues: initialData,
+    resolver: yupResolver(baseImageCollectionSchema),
+  });
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Stack gap={3}>
+        <FormRow
+          id="base-image-collection-form-name"
```
Comment (davidebriani) [inline frontend/src/forms/CreateBaseImageCollection.tsx:102]: I'd say that these IDs should be prefixed with *create*: `id="create-base-image-collection-form-name"`

**Review by davidebriani — CHANGES_REQUESTED** [review]

**Pull request merged**